### PR TITLE
XOR-412 [Fix] Ensure Send button is visible on mobile for Chat Screen

### PR DESCRIPTION
**Pull request by NikitaJadhavXoriant**
### Product areas affected

Widgets -> Chat Widget  -> Select Contact from List to chat -> Send

### What does this PR do?

Implemented fix so that visibility issue of Send button on mobile for Chat Screen is resolved

### JIRA ticket

[JIRA](https://weboo.atlassian.net/browse/XOR-412)

### Result

[XOR-Test-412.webm](https://user-images.githubusercontent.com/108272606/188843081-b0c6ad5f-3812-46e7-aeb6-26d95b7df508.webm)

### Checklist

none

### Testing instructions

none

### Deployment instructions

none

### Author concerns

none

### CODE REVIEW

**Review by amolsuryawanshi47 — APPROVED** [review]
looks ok to me

File: css/chat.css [hunk 1]
```diff
@@ -18,6 +18,12 @@ html.no-supports-container.native body,
   height: calc(100vh - 44px);
 }
 
+@media screen and (max-width: 640px) {
+  .supports-container .fl-with-bottom-menu .fl-widget-instance[data-name="Chat"] {
```
Comment (tonytlwu) [inline css/chat.css:22]: Let's change `[data-name="Chat"]` to use `data-package` instead as it's less likely to change over time

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: css/chat.css (hunk 1 above)
Comment (NikitaJadhavXoriant) [reply css/chat.css:22]: Should I change all the occurrences(7 places) in chat.css file to [data-package="com.fliplet.chat"]

Comment (tonytlwu) [reply css/chat.css:22]: @NikitaJadhavXoriant Yes please

Comment (NikitaJadhavXoriant) [reply css/chat.css]: Changes done

**Review by amolsuryawanshi47 — APPROVED** [review]
Looks ok to me

**Review by tonytlwu — APPROVED** [review]

**Pull request merged**